Libs/Python/BiochemPy/Compounds.py

```diff
@@ -60,10 +62,29 @@ def loadSourceAliases(self):
             if(line['External ID'] not in aliases_dict[line['Source']]):
                 aliases_dict[line['Source']][line['External ID']]=list()
 
-            aliases_dict[line['Source']][line['External ID']].append(line['MS ID'])
+            aliases_dict[line['Source']][line['External ID']].append(line['ModelSEED ID'])
 
         return aliases_dict
 
+    def loadNames(self):
```

Comment:
These function is entire duplicate (except for the variable names and file path). Why not just make one function that accepts the path to the alias/name file?

Reply:
Thanks for checking, and yes, I went back and forth on this, but I eventually decided to separate them out, and EC numbers/classes because I can see the names and ECs being edited at a higher rate, and in different ways.

We won't edit external identifiers, they'll be considered immutable to a degree, but we will change associated cpd/rxn identifiers. On the other hand, we'll likely directly edit many of the actual names (i.e. upper/lower case etc.).

ECs are a mess in their own right, with little to no provenance, so I'm anticipating that we'd start over somehow with their original source at Expasy.

Ultimately, for files that may get edited a lot, we'll see a lot of conflicts when merging, and I'm hoping that separating our the different kinds of aliases will reduce some of the confusion.